package.json

```diff
@@ -178,7 +178,7 @@
     "jest-raw-loader": "^1.0.1",
     "matrix-mock-request": "^1.2.3",
     "matrix-react-test-utils": "^0.2.3",
-    "matrix-web-i18n": "github:matrix-org/matrix-web-i18n",
+    "matrix-web-i18n": "github:matrix-org/matrix-web-i18n#a8a01807c057595948a6da3587516c46df77de18",
```

Comment:
I'm a bit concerned that the version will be fixed to a specific commit. Whereas before it would get automatically updated whenever something is pushed to the `main` branch.

Sometimes we might need to do `yarn cache clean` before re-installing the dependencies to make sure that `yarn.lock` gets updated with the correct revision

Reply:
true, removing